supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -363,6 +364,12 @@ Always use the approved product name _Red Hat build of OpenJDK_ instead.
 [[brms]]
 ==== image:images/yes.png[yes] Red Hat JBoss BRMS (noun)
```

Comment:
Just want to check that the "Red Hat JBoss BRMS" is still required? Just nopticed that Git has added the full entry in the comparison tool.

Reply:
Not required. I'll check the source.